api/swagger.yaml

```diff
@@ -7584,6 +7584,11 @@ paths:
           description: "ID or name of service."
           required: true
           type: "string"
+        - name: "insertDefaults"
```

Comment:
This needs to be added to the API version-history; https://github.com/docker/docker/blob/master/docs/api/version-history.md

(ok with doing that in a follow-up)